httpx/config.py

```diff
@@ -8,9 +8,18 @@
 from .__version__ import __version__
 from .utils import get_ca_bundle_from_env, get_logger
 
+
+class UnsetType:
+    pass
+
+
+UNSET = UnsetType()
```

Comment:
Had to put this here instead of the model package due to a circular dependency.

Reply:
`config.py` is probably a nicer home for this constant anyway. :)